### ISSUE TITLE
Composer Private Endpoint should have the "Global access" (multi-region) option 

### ISSUE THREAD

**Issue by lassebenni** [terraform-google-modules/terraform-google-composer#30]
### TL;DR

Please add an option to enable private, but global (cross-region) access to the underlying kubernetes cluster that composer creates.

### Terraform Resources
When creating a composer environment, it's possible to set the endpoint to private:

``` terraform
module "composer" {
  source = "terraform-google-modules/composer/google//modules/create_environment"

  project = "project-123"
  name    = "Composer-Prod-Env"
  region  = "eu-west1"
  ...
   enable_private_endpoint = True
}

```

Behind the scenes, this will create a Kubernetes cluster with a private endpoint to the master control plane. So far so good. However, by default this endpoint is **ONLY available from the same region** as the GKE cluster. Trying to run `gcloud composer` commands from a VM in another region will fail:

``` bash
kubeconfig entry generated for europe-west1-composer-123456-gke.
Unable to connect to the server: dial tcp 172.17.8.2:443: i/o timeout
ERROR: (gcloud.composer.environments.run) kubectl returned non-zero status code.
 
```

This caused quite a headache (trying out all sorts of networking options, is it firewalling, is it routing etc.) until we discovered that the `google_container_cluster` has the `master_global_access_config` argument that is [False by default, AND not tracked by Terraform.](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#nested_master_global_access_config)

We manually enabled the setting, but it would've saved a lot of time if the option was available when creating the `google_composer_environment` in the first place.
